justfile

```diff
@@ -6,16 +6,16 @@ BINARYEN_CHECKSUM := "c55b74f3109cdae97490faf089b0286d3bba926bb6ea5ed00c8c784fc5
 default:
     just --list
 
+# TODO: reenable this   cd odra-casper/proxy-caller && cargo clippy --target=wasm32-unknown-unknown -- -D warnings -A clippy::single-component-path-imports
 clippy:
     cargo clippy --all-targets -- -D warnings
```

Comment:
This should be rewritten, as examples, modules and benchmarks are now in workspace, so one call should be enough.